utils/index.sh

```diff
@@ -71,23 +71,27 @@ setup(){
 }
 
 get_ipsec_config(){
-    # Get a ovnkube-master pod to try to ge the ipsec value
     ipsec=false
-    ovn_pod=""
-    if result=$(oc get pods -o custom-columns=name:.metadata.name -n openshift-ovn-kubernetes --no-headers=true | grep ovnkube-master -m1); then
-        ovn_pod=$result
-    fi
-    if [[ -z "$ovn_pod" ]]; then
-        if result=$(oc get pods -o custom-columns=name:.metadata.name -n openshift-ovn-kubernetes --no-headers=true | grep ovnkube-node -m1); then
-            ovn_pod=$result
-        fi
-    fi
-
-    # Check if the pod has the container nbdb, if it is OVNIC it won't
-    # If it is a OVN the command will succed and the result will be stored in ipsec
-    if result=$(oc -n openshift-ovn-kubernetes -c nbdb rsh $ovn_pod ovn-nbctl --no-leader-only get nb_global . ipsec); then
-        if [[ $result == *"true"* ]]; then
-            ipsec=true
+    ipsecMode="Disabled"
+    if result=$(oc get networks.operator.openshift.io cluster -o=jsonpath='{.spec.defaultNetwork.ovnKubernetesConfig.ipsecConfig.mode}'); then
+        # If $result is empty, it is version older than 4.15
+        # We need to chekc a level above in the jsonpath
+        # If that level is not empty it means ipsec is enabled
+        if [[ -z $result ]]; then
+            if deprecatedresult=$(oc get networks.operator.openshift.io cluster -o=jsonpath='{.spec.defaultNetwork.ovnKubernetesConfig.ipsecConfig}'); then
+                if [[ ! -z $deprecatedresult ]]; then
+                    ipsec=true
+                    ipsecMode="Full"
+                fi
+            fi
+        else
+            # No matter if enabled and then disabled or
+            # Disabled by default
+            # this field is always shows Disabled when no IPSec
```

Comment:
rephrase please?

Reply:
Any suggestions, on how to explain the situation

Reply:
```suggestion
            # No matter if Enabled and then Disabled or Disabled by default, this field always shows Disabled when no IPSec.
```